python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -177,14 +177,19 @@ def _to_int(value):
 # I wanted to use default=None here to match Python API but click silently
 # replaces None with an empty tuple when multiple=True.
 qgraph_node_id_option = MWOptionDecorator("--qgraph-node-id",
-                                          callback=_split_commas_int,
+                                          callback=split_commas,
```

Comment:
If this isn't an integer anymore (hex?) you should also change the description in the help two lines down to say what it is.